src/epgsql_sock.erl

```diff
@@ -298,6 +298,13 @@ terminate(_Reason, #state{mod = ssl, sock = Sock}) -> ssl:close(Sock).
 code_change(_OldVsn, State, _Extra) ->
     {ok, State}.
 
+format_status(normal, [_PDict, State=#state{}]) ->
+  [{data, [{"State", State}]}];
+format_status(terminate, [_PDict, State]) ->
+  %% Do not format the rows attribute when process terminates abnormally
+  %% but allow it when is a sys:get_status/1.2
+  State#state{rows = information_redacted}.
```

Comment:
In the shell, there's some kind of thing to print a term... but only up to a certain size. I forget what it is, but maybe something like that could be used to print a bit, but truncate it?

Reply:
We introduced a similar tuning in our system that limits the size up to 50-100 but we did not have this in place for many many years until the last incident.
Hence, let's try to be defensive and not assume all the systems that use `epgsql` have this kind of tuning in place.